hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestOMMultiTenantManagerImpl.java

```diff
@@ -66,7 +66,7 @@ public void setUp() throws IOException {
     conf = new OzoneConfiguration();
     conf.set(OZONE_OM_DB_DIRS, folder.toAbsolutePath().toString());
     conf.set(OZONE_OM_TENANT_DEV_SKIP_RANGER, "true");
-    omMetadataManager = new OmMetadataManagerImpl(conf, ozoneManager);
+    omMetadataManager = new OmMetadataManagerImpl(conf, ozoneManager, new OMPerformanceMetrics());
```

Comment:
For unit tests that don't care about metrics, can we please avoid the change?  `OmMetadataManagerImpl` itself could create a dummy metrics object.  Or it could be changed to be able to handle `null`.

Reply:
Noted, kept it same as earlier by changing approach.